src/Uno.UI/Extensions/ViewHelper.iOSmacOS.cs

```diff
@@ -224,48 +225,17 @@ public static nfloat StackSubViews (_View thisView, float topPadding, float spac
 		/// Gets the orientation-dependent screen size
 		/// </summary>
 		/// <returns></returns>
-		public static CGSize GetScreenSize()
```

Comment:
Isn't this going to be a breaking change?

Reply:
![NowThatYouMentionItTylerOakleyGIF](https://user-images.githubusercontent.com/1075116/168027667-1d41aaa3-6de9-4eee-8700-94e0850c88c1.gif)